### PR TITLE
latexmk.md: add page

### PR DESCRIPTION
**Pull request by Nagiza**
Described most used and forgotten latexmk commands.

### CODE REVIEW

**Comment by agnivade** [comment]
@sbrl - tldr bot seems to be down again. Can you check ?

**Comment by sbrl** [comment]
Ouch! Yeah, it does appear to have died again. This time it looks like `libssl1.0.2`, which uswgi depends on, was removed automatically during an update:

```
/usr/local/bin/uwsgi: error while loading shared libraries: libssl.so.1.0.2: cannot open shared object file: No such file or directory
```

That's no good! I've reinstalled `libssl1.0.2` manually, and started the job again. I've also configured the server to automatically restart the tldr-bot at the dead of night every day:

```crontab
0 0 * * *   /bin/systemctl restart tldr-bot.service
```

It should be working again now. If there are any more problems with it, just let me know :smiley_cat:

File: pages/linux/latexmk.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# latexmk
```
Comment (sbrl) [inline pages/linux/latexmk.md]: Thanks for adding this! It looks useful.

File: pages/linux/latexmk.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+> Automated LaTeX document generation
+
+ - Compile .tex file to pdf:
+
+`latexmk -pdf file`
+
+ - Force latexmk to generate document despite errors:
+
+`latexmk -f file`
```
Comment (sbrl) [inline pages/linux/latexmk.md]: There isn't a long version of this switch, is there?

Comment (Nagiza) [inline pages/linux/latexmk.md]: No, there is only this one letter option.

File: pages/linux/latexmk.md (hunk 1 above)
Comment (Nagiza) [inline pages/linux/latexmk.md]: Thanks, it's a lot easier than most alternatives.

File: pages/linux/latexmk.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# latexmk
+
+> Automated LaTeX document generation.
+
+- Compile .tex file to pdf:
+
+`latexmk -pdf file`
```
Comment (agnivade) [inline pages/linux/latexmk.md]: Suggest changing file to file.tex. And please wrap it in tokens.

File: pages/linux/latexmk.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Force latexmk to generate document despite errors:
+
+`latexmk -f file`
+
+- Clean up files generated by latexmk except dvi, postscript and pdf:
```
Comment (agnivade) [inline pages/linux/latexmk.md]: I am guessing it cleans up files in the current folder ? Would be good to mention it for clarity's sake.

File: pages/linux/latexmk.md (hunk 3 above)
Comment (sbrl) [inline pages/linux/latexmk.md]: My, I keep forgetting the tokens! Guess I'm too busy looking at the grammar & readability.... I'll have to try and remember them next time :P

File: pages/linux/latexmk.md (hunk 4 above)
Comment (sbrl) [inline pages/linux/latexmk.md]: Ah, good point

**Comment by sbrl** [comment]
:smile_cat:

**Pull request merged**